scripts/tests/test_suites.sh

```diff
@@ -134,6 +134,11 @@ for j in "${iter_array[@]}"; do
         # kicking off the subshell, sometimes we try to do it before
         # the data is there yet.
         background_pid="$(</tmp/pid)"
+        if command -v dns-sd &>/dev/null; then
```

Comment:
Perhaps document that dns-sd is not available in all the situations where we run this script, and we just want to run it if it is?